README.md

```diff
@@ -39,6 +40,16 @@ selected service should be accessible on port :8080 or :8081:
  - Signing Service: http://localhost:8080/v1/version
  - Admin Service: http://localhost:8081/
 
+NOTE: Admin service csrf protection enforces accessing the service through HTTPS and using a valid
```

Comment:
Let's change the wording here:
1. Always capitalise CSRF (it's a title, like HTTPS and SSO).
2. CSRF does not enforce the service through HTTPS, it is just that the cookies are not sent unless HTTPS is used. The workaround for this is setting the environment variable.
3. I don't think you need to have "NOTE:" here, it reads fine without it.

It would be worth reading the docs on the Secure() method as that indicates that it sets the "secure" flag on the cookie, which means that it will only be sent over an HTTPS connection. Disabling this feature with the environment variable ensures that cookies are also sent over HTTP - https://github.com/gorilla/csrf/blob/master/options.go#L45